docs/interactions/ui-components/dropdowns.mdx

```diff
@@ -100,6 +102,39 @@ In the `options` parameter, you pass a list of [`discord.SelectOption`](https://
 -   `label` (the name displayed to users, can be up to 100 characters)
 -   `value` (a special value of the option, defaults to the label).
 
+## Select Types
+
+In addition to regular string selects, you can also have your select menu contain users, roles, mentionables, and channels as its options. You can use these alternative select types by passing a [`discord.ComponentType`](https://docs.pycord.dev/en/stable/api/enums.html#discord.ComponentType) value for the `select_type` parameter when creating the Select.
+
+:::info Viewing the selected option
+
+When accessing the value the user selected through `select.values[0]`, you will only be able to retrieve the ID of the user, role, mentionable, or channel if you are not using a [`discord.ComponentType.string_select`](https://docs.pycord.dev/en/stable/api/enums.html#discord.ComponentType.string_select).
+
+:::
```

Comment:
This is incorrect, `Select.values` will resolve the received values to their respective objects, as they are sent with the interaction when selected.
https://github.com/Pycord-Development/pycord/blob/master/discord/ui/select.py#L328
https://docs.pycord.dev/en/master/api/ui_kit.html#discord.ui.Select.values